javascript/javascript2/week1/hbay/main.js

```diff
@@ -0,0 +1,37 @@
+console.log("Script loaded");
+
+function renderProducts(arr) {
+  const mainUl = document.querySelector("section.products ul");
+  arr.forEach((product) => {
+    const productsLi = document.createElement("li");
+    const productsUl = document.createElement("ul");
```

Comment:
Product Ul is the parent of all and needed once, therefor should be outside forEach.